test/unit/io/sam_file/sam_file_input_test.cpp

```diff
@@ -406,17 +421,19 @@ std::string input_bgzf
 
 TEST_F(sam_file_input_f, decompression_by_filename_bgzf)
 {
-    seqan3::test::tmp_filename filename{"sam_file_output_test.sam.bgzf"};
+    seqan3::test::tmp_directory tmp{};
+    auto filename = tmp.path() / "sam_file_onput_constructor.sam.bgzf";
```

Comment:
I wonder what an "onput_constructor" is

Reply:
oh no, yes that is not correct. The name doesn't matter that much, but I will correct it.